eve/main.yml

```diff
@@ -110,8 +110,8 @@ models:
 
   - env: &k8s_env # containes values for running helm and kubectl
       ZENKO_HELM_RELEASE: 'zenko-test'
-      HELM_NAMESPACE: '%(prop:testNamespace)s'
-      TILLER_NAMESPACE: '%(prop:testNamespace)s'
+      HELM_NAMESPACE: '%(prop:workername)s'
+      TILLER_NAMESPACE: '%(prop:workernamee)s'
```

Comment:
typo :')

Reply:
this is why I hate using my laptop keyboard. Thank you!